main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+// http://blog.sgmansfield.com/2016/01/the-hidden-dangers-of-default-rand/
```

Comment:
Thanks for the info! For now, I'll keep the `rand()` as it is since the whole program is really small.